src/CallObjectLoader.js

```diff
@@ -159,70 +159,165 @@ class LoadAttempt {
     this.cancelled = false;
     this.succeeded = false;
 
-    this._timedOut = false;
-    this._timeout = null;
+    this._networkTimedOut = false;
+    this._networkTimeout = null;
+
+    this._iosCache =
+      typeof iOSCallObjectBundleCache !== "undefined" &&
```

Comment:
I think this check may be unnecessary.  `iOSCallObjectBundleCache` could be `null` which would make `this._iosCache` `null` but AFAICT the code flow still works.

Reply:
Ah. The problem is that if `iOSCallObjectBundleCache` doesn't exist, it won't simply be `null`, it'll be an undefined variable (meaning the code will crash if we try to access it). It's set on the global scope by `react-native-daily-js` when it detects we're running on iOS (the global scope unfortunately being the only way to manage communication between code running in `daily-js` and the call bundle it eventually loads).

Reply:
Let me know if I misunderstood your comment!

Reply:
got it. and I assume checking `typeof` is safer than a straight `iOSCallObjectBundleCache !== undefined` ?

Reply:
That would crash too :/ Can't reference a variable that hasn't been defined.

What *wouldn't* crash is `window.iOSCallObjectBundleCache !== undefined` (since it's not referencing an undefined variable, but a property of something which is defined), but I didn't want to add more references to `window` since we're trying to write React Native-friendly code. `window` *happens* to exist since we use it for some React Native shims, but I'd like to avoid further baking in that assumption in more places than is necessary, in case it changes someday.

Reply:
👍